packages/babel-parser/src/util/location.ts

```diff
@@ -6,14 +6,15 @@ export type Pos = {
 // `startLoc` and `endLoc` properties.
 
 export class Position {
+  // If `declare` is used here, the `jquery` benchmark will drop by 10%. :)
   line: number;
   column: number;
-  index: number;
+  declare index?: number;
```

Comment:
Ugh, this is super weird, I tried for a long time before I found it was related to it.🤦‍♂️